eptest/eptest.go

```diff
@@ -79,3 +82,19 @@ func BenchWithContext(ctx context.Context, r ep.Runner, datasets ...ep.Dataset)
 
 	return err
 }
+
+// ErrRunner is a Runner that returns an error upon first input or inp closing
+type ErrRunner struct {
```

Comment:
please make it private and expose `eptest.NewErrRunner(e error) ep.Runner` function
```suggestion
type errRunner struct {
```